lib/large_object_store.rb

```diff
@@ -64,12 +73,7 @@ def read(key)
         data = Zlib::Inflate.inflate(data)
       end
 
-      begin
-        Marshal.load(data)
-      rescue Exception => e
-        Rails.logger.error "Cannot read large_object_store key #{key} : #{e.message} #{e.backtrace.inspect}"
-        nil
-      end
+      Marshal.load(data)
```

Comment:
since we can guarantee proper cache handling and no invalid pages this should never happen ...